test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -150,15 +147,18 @@ func TestSchedule1000Node30KPods(t *testing.T) {
 // 	}
 // }
 
+// testConfig contains the some input parameters needed for running test-suite
 type testConfig struct {
-	numPods                   int
-	numNodes                  int
+	// Note: We don't need numPods, numNodes anymore in this struct but keeping them for backward compatibility
```

Comment:
So, the idea is to push these values into schedulerPerfConfig struct, so that there will be a single entry point into code base. Once we have them, we can remove them from the testConfig struct.(This will have fields that are specific to internal data representation like nodeStrategies etc.